playbooks/install/satellite_63.yml

```diff
@@ -1,4 +1,10 @@
 ---
 - hosts: sat63
+  vars_files:
+    - ../../vars/satellite_common.yml
```

Comment:
This should be `../../vars/satellite_common_local.yml` which would not be added to version control. Will fix these.

Need to add the workflow to the README file about, `cp satellite_common.yml satellite_common_local.yml`

Reply:
One good thing would be to add `satellite_common_local.yml` to `.gitignore`  to avoid any accidental addition of it.

Reply:
populating `satellite_common_local.yml` would just be a one-time effort.